sims/SADDsims_FullArray_001/README.txt

```diff
@@ -0,0 +1,9 @@
+The simulation in this directory aims to provide quantitative values for some of the metrics relevant for the System Architecture Description Document (SADD).  Specifically, it quantifies the "minimum baseline RMS noise," "inverse shortest projected non-intrasite baseline," "inverse longest projected baseline," and "point source sensitivity" metrics appropriate for "full array" operations of the array.
+
+To set up the necessary environment for running the simulation, the user must first install the Python-based ngehtsim package.  The original metric calculations were run using ngehtsim version 1.0.0.  Instructions for installing the package can be found on the ngehtsim github repository (https://github.com/Smithsonian/ngehtsim).  Installation of ngehtsim will also install all required dependencies for the simulation.
```

Comment:
From one of my earlier emails:

> If the intent is to simply run these scripts from the directory they live in within the cloned repository, then I think all you really need is a requirements.txt that contains a line like `ngehtsim==1.2.3` and some text in your README that says (a) what version of Python you need and (b) to run `pip install -r requirements.txt`.

Since ngehtsim isn't published in PyPI, `requirements.txt` should contain
```
ngehtsim @ git+https://github.com/Smithsonian/ngehtsim.git@v1.0.0
```

This applies to `SADDsims_MonitoringArray_001` as well.